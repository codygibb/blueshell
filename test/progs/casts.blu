i := 123
f := 1.23
s := "123"

print (float) (i * 92)
print (string) (i * 92) + " is the answer"
print (string) true + " should be true"
if ((bool) "true") {
  print "this should be printed"
} else {
  print "this should not be printed"
}
print (int) f

# test precedence
print (float) i * f

