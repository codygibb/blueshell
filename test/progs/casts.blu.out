11316.000000
11316 is the answer
true should be true
this should be printed
1
151.290000
